spec/controllers/admin/posts_controller_spec.rb

```diff
@@ -44,10 +44,15 @@
 
   describe 'POST create' do
     describe 'with valid params' do
-      it 'creates a new Admin::Post' do
-        expect {
-          post :create, post: valid_attributes
-        }.to change(Post, :count).by(1)
+      let(:file1) { Rack::Test::UploadedFile.new('spec/support/1.jpg', 'image/jpeg') }
```

Comment:
Line is too long. [86/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.